packages/webapp/src/routes/developer/upload-metric-file.js

```diff
@@ -26,7 +26,7 @@ const performUpload = async (request, h) => {
       request.yar.set(constants.redisKeys.DEVELOPER_METRIC_DATA, metricFileData[0].metricData)
       request.yar.set(constants.redisKeys.DEVELOPER_METRIC_LOCATION, metricFileData[0].location)
       request.yar.set(constants.redisKeys.DEVELOPER_METRIC_FILE_NAME, metricFileData.filename)
-      request.yar.set(constants.redisKeys.DEVELOPER_METRIC_FILE_SIZE, parseFloat(metricFileData.fileSize).toFixed(1))
+      request.yar.set(constants.redisKeys.DEVELOPER_METRIC_FILE_SIZE, parseFloat(metricFileData.fileSize).toFixed(2))
```

Comment:
This increase in the number of decimal places will need retaining when the common upload code mentioned by @teddmason elsewhere on this pull request is incorporated. If the number of decimal places is also to be increased for the developer metric I suggest making that small change on this branch when the common code is incorporated rather than creating another story to make such a small change.

Reply:
We need to increase the number of decimal places as if the file size is too small like 0.03 MB in that case if we are showing only one decimal by roundoff it shows a file size of 0.0 MB.